### PR TITLE
editing configurations and editing rules in feature flags

### PR DESCRIPTION
**Pull request by jenniferopal**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \ creating pages on the edit config and the edit rules permissions.
* Jira/GitHub Issue numbers (if any): \ FFM-11328
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [X] Successful preview build.
- [X] Code owner review.
- [X] No merge conflicts.
- [X] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/feature-flags/ff-creating-flag/editing-rules-and-config.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: Managing rules of your flags
+description: This page discusses using Harness Feature Flags with Cloud Cost Module and how it's used. 
+sidebar_position: 35
+---
+
+## Managing rules of your flags
+
```
Comment (SmritiSatya) [inline docs/feature-flags/ff-creating-flag/editing-rules-and-config.md:8]: Manage flag rules

File: docs/feature-flags/ff-creating-flag/editing-rules-and-config.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+### Editing rules
+
```
Comment (SmritiSatya) [inline docs/feature-flags/ff-creating-flag/editing-rules-and-config.md:10]: Edit rules

File: docs/feature-flags/ff-creating-flag/editing-rules-and-config.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+Ensure the flag is turned on for your selected environment for your changes to run successfully. 
+
+When editing your rules, you may have noticed the option to add **Targets** and, or, **Target Groups**. If you'd like to learn more on how to create **Targets** and **Target Groups**, have a look at some useful Harness docs on the topic:
+
+ * [Add Targets](/docs/feature-flags/ff-target-management/add-targets.md)
+ * [Add Target Groups](/docs/feature-flags/ff-target-management/add-target-groups.md)
+ * [Targeting Users with Flags](/docs/feature-flags/ff-target-management/targeting-users-with-flags.md)
+
+## Editing configuration
+
```
Comment (SmritiSatya) [inline docs/feature-flags/ff-creating-flag/editing-rules-and-config.md:33]: Edit configuration

File: docs/feature-flags/ff-creating-flag/using-ff-ccm.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Features Flag with Cloud Cost Management Overview
+title: Using Cloud Cost Management with your flags
 description: This page discusses using Harness Feature Flags with Cloud Cost Module and how it's used.
```
Comment (SmritiSatya) [inline docs/feature-flags/ff-creating-flag/using-ff-ccm.md:3]: Use ...

File: docs/feature-flags/ff-creating-flag/editing-rules-and-config.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+title: Managing rules of your flags
+description: This page discusses using Harness Feature Flags with Cloud Cost Module and how it's used. 
+sidebar_position: 35
+---
+
+## Managing rules of your flags
+
+### Editing rules
+
+Within Harness Feature Flags, rules contains the targeting logic within your flags. The rules help describe what variation the flag should present to the user when the flag is enabled or disabled.
```
Comment (SmritiSatya) [inline docs/feature-flags/ff-creating-flag/editing-rules-and-config.md:11]: Within Harness Feature Flags, rules contain ...

File: docs/feature-flags/ff-creating-flag/editing-rules-and-config.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+### Editing rules
+
+Within Harness Feature Flags, rules contains the targeting logic within your flags. The rules help describe what variation the flag should present to the user when the flag is enabled or disabled. 
+
+When you create a Feature Flag within your project, you have the option to specify a specific variation using **Default Rules**. 
+
+ ![Editing default rules to specify a variation](./static/ff-editing-rules-1.png)
+
+In order to do this, you would need to:
```
Comment (SmritiSatya) [inline docs/feature-flags/ff-creating-flag/editing-rules-and-config.md:17]: To do this, ...

File: docs/feature-flags/ff-creating-flag/editing-rules-and-config.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+ 1. Go into your **Project** and select **Feature Flags** on the left hand menu panel.
```
Comment (SmritiSatya) [inline docs/feature-flags/ff-creating-flag/editing-rules-and-config.md:19]: Go to ...

File: docs/feature-flags/ff-creating-flag/editing-rules-and-config.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+Within Harness Feature Flags, rules contains the targeting logic within your flags. The rules help describe what variation the flag should present to the user when the flag is enabled or disabled. 
+
+When you create a Feature Flag within your project, you have the option to specify a specific variation using **Default Rules**. 
+
+ ![Editing default rules to specify a variation](./static/ff-editing-rules-1.png)
+
+In order to do this, you would need to:
+
+ 1. Go into your **Project** and select **Feature Flags** on the left hand menu panel. 
+ 2. If editing the rule of an existing flag, double click on your chosen flag. Otherwise, go through the steps to create a new one and you may continue with the next step below.
```
Comment (SmritiSatya) [inline docs/feature-flags/ff-creating-flag/editing-rules-and-config.md:20]: Global:
double-click

File: docs/feature-flags/ff-creating-flag/editing-rules-and-config.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+ 3. On the right hand side of the page, the **Targeting** option should already be selected and you can proceed to editing your rules for when the flag is enabled or when the flag is disabled.
```
Comment (SmritiSatya) [inline docs/feature-flags/ff-creating-flag/editing-rules-and-config.md:21]: On the right-hand side of the page, the **Targeting** option should already be selected and you can proceed to edit your rules for when the flag is enabled or when the flag is disabled.

File: docs/feature-flags/ff-creating-flag/editing-rules-and-config.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+In order to do this, you would need to:
+
+ 1. Go into your **Project** and select **Feature Flags** on the left hand menu panel. 
+ 2. If editing the rule of an existing flag, double click on your chosen flag. Otherwise, go through the steps to create a new one and you may continue with the next step below.
+ 3. On the right hand side of the page, the **Targeting** option should already be selected and you can proceed to editing your rules for when the flag is enabled or when the flag is disabled. 
+ 4. Once you have made the changes you need, select **Save** at the bottom of the screen.  
+
+Ensure the flag is turned on for your selected environment for your changes to run successfully. 
+
+When editing your rules, you may have noticed the option to add **Targets** and, or, **Target Groups**. If you'd like to learn more on how to create **Targets** and **Target Groups**, have a look at some useful Harness docs on the topic:
```
Comment (SmritiSatya) [inline docs/feature-flags/ff-creating-flag/editing-rules-and-config.md:26]: To learn more on ..., go to :

File: docs/feature-flags/ff-creating-flag/editing-rules-and-config.md [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+
+## Editing configuration
+
+When setting the rules of your flag, you have the options to add additional configurations to further monitor the performance of the Feature Flags as a user or for your team. 
+
+Here are the steps to do so: 
+
+ 1. Go into your **Project** and select **Feature Flags** on the left hand menu panel. 
+ 2. If editing the rule of an existing flag, double click on your chosen flag. Otherwise, go through the steps to create a new one and you may continue with the next step below.
+ 3. On the left hand panel underneath the name of your Feature Flag, you will find options to add **Cloud Cost Monitoring**, **Monitored Services**, and **Jira Issues**.
```
Comment (SmritiSatya) [inline docs/feature-flags/ff-creating-flag/editing-rules-and-config.md:40]: left-hand

**Review by SmritiSatya — DISMISSED** [review]

**Review by crystal-fong — APPROVED** [review]

**Pull request merged**